lib/resources/r4/care_team.yaml

```diff
@@ -0,0 +1,48 @@
+---
+name: CareTeam
+field_name_base_url: http://hl7.org/fhir/R4/careteam-definitions.html#CareTeam
+fields:
+- name: patient
+  required: 'Yes'
+  type: Reference
+  description: Indicates the patient who the sensitivity is for.
+  example: |
+    {
+      "patient": {
+        "reference": "Patient/631923"
+      }
+    }
+  url: https://hl7.org/fhir/r4/careteam-definitions.html#CareTeam.patient
+
+- name: encounter
```

Comment:
encounter doesn't appear to be needed in your .md

Reply:
Encounter field is needed. Made the changes as per requirement.

Reply:
this yaml file is consumed by various tables embedded in the markdown files. the only table I see in markdown is a terminology bindings table. since encounter is not bound to any terminology (as it is a reference field, not a coded field), I don't understand why it needs to be here

am i missing anything?

Reply:
+1. Nimisha you can remove the fields which are not present in terminology bindings

Reply:
Removed. https://github.com/cerner/fhir.cerner.com/pull/470/commits/60de10bf8259905011fb638e99b4f1607031c126#diff-c37b2a12be12c28b23605fef2eacc3ceL50-L62